week-1/H-numbers/exercise.js

```diff
@@ -1 +1,10 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 15,
+  numberOfMentors = 8;
```

Comment:
You should use `var` for declaring variables. Otherwise Javascript treats them as global.

Reply:
I could use var but when I pressed the Ctrl + S VS Code changed to look like that. 
The other one thing You can use var on the one line for declaring more than one variables.

Reply:
You're right. I didn't notice you were using commas `,`, instead of semicolons `;` :slightly_smiling_face: .